apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -107,6 +108,8 @@ function getParamsForAssessment(
   const assessmentCanSubmit = assessment.canSubmit.length > 0 ? assessment.canSubmit : allRoleNames;
   const alternativeGroups = assessment.zones.map((zone) => {
     const zoneGradeRateMinutes = zone.gradeRateMinutes ?? assessment.gradeRateMinutes ?? 0;
+    const zoneAllowRealTimeGrading =
```

Comment:
I assume this is out of date, because I thought we weren't dealing with defaults in this code anymore.

Reply:
I am referring to that.